src/lib/shared/ui/snackbar/snackbar.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { render } from '@testing-library/svelte';
```

Comment:
Shouldn't this file be under s folder called `specs`?

Reply:
Why should it be there? FSD doesn't require the `specs` folder

Reply:
In other parts of this project we put segment tests into the `specs` folder inside each segment. Just for consistency with the rest of the project